health/micro-ui/web/micro-ui-internals/example/src/UICustomizations.js

```
@@ -1222,7 +1221,7 @@ export const UICustomizations = {
 
       switch (key) {
         case "MICROPLAN_FACILITY_ASSIGNED_VILLAGES":
-          const assignedVillages = row?.additionalDetails?.assignedVillages;
+          const assignedVillages = row?.serviceBoundaries;
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)